entrypoint.sh

```diff
@@ -4,10 +4,11 @@ echo ${KUBE_CONFIG_DATA} | base64 -d > kubeconfig
 export KUBECONFIG=kubeconfig
 
 helm plugin install https://github.com/zendesk/helm-secrets
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
@mingujotemp Should the stable repo be added inside the Dockerfile and not inside the entrypoint? 

I was already trying to move the `helm plugin install...` inside the Dockerfile so wondered if we could hit two birds with one stone?

Otherwise LGTM, I'll merge!